my-app/src/Components/History/DetailHistory.jsx

```diff
@@ -0,0 +1,169 @@
+import { useState, useEffect } from "react";
+import {
+  useLocation,
+  useParams,
+  useNavigate,
+  useSearchParams,
+} from "react-router-dom";
+import {
+  getAvgByCheckpointId,
+  getReviewsByCheckpointId,
+} from "../../Api/userApi";
+import "./DetailHistory.css";
+function DetailHistory() {
+  const navigate = useNavigate();
+  const params = useParams();
+  const search = useLocation().search;
+  const page = new URLSearchParams(search).get("page") || 1;
+
+  const [searchParams] = useSearchParams();
+  const title = searchParams.get("title");
+  const itemsPerPage = 10;
+  const start = (page - 1) * itemsPerPage;
+  const end = page * itemsPerPage;
+  const [listReviews, setListReviews] = useState([]);
+  const [dataPerPage, setDataPerPage] = useState([]);
+  const [dataAvg, setDataAvg] = useState({
+    avg_attitude: 0,
+    avg_performance: 0,
+    avg_teamwork: 0,
+    avg_training: 0,
+    avg_adhere: 0,
+  });
+  let [numPages, setNumPages] = useState(1, []);
```

Comment:
dùng const, vì sao ở đây em sử dụng 
```useState(1, [])```

Reply:
numPage này, em để giá trị defatult là 1, sau khi call Api, tính toán lại số trang thì thay đổi số lượng page nên em dùng useState(), bình thường em dùng const, vậy em sửa lại dùng const nha